releaf-core/app/assets/stylesheets/releaf/include/error.scss

```diff
@@ -16,18 +14,17 @@ section.error h1 i
     color: #ef3d64;
     font-size: 25px;
     position: absolute;
-    left: -62px;
-    top: 14px;
+    left: 0px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>`0px` should be written without units as `0`